modules/java/HTTPDestination.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Created on 6/29/15.
+ */
+import org.syslog_ng.*;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+
+public class HTTPDestination extends TextLogDestination {
+
+    private String name;
+    private URL url;
+
+    public HTTPDestination(long arg0) {
+        super(arg0);
+    }
+
+    public void deinit() {
+    }
+
+    public void onMessageQueueEmpty() {
+    }
+
+    public boolean init() {
+        this.name = getOption("name");
+        try {
+            this.url = new URL(getOption("url"));
+        } catch (MalformedURLException e) {
+            InternalMessageSender.error("A properly formatted URL is a required option for this destination");
+            return false;
+        }
+        if (name == null) {
+            InternalMessageSender.error("A unique name is a required option for this destination");
+            return false;
+        }
+
+        InternalMessageSender.debug("Init " + name);
+        return true;
+    }
+
+    public boolean open() {
+        return true;
+    }
+
+    public boolean isOpened() {
+        return true;
+    }
+
+    public void close() {
+
```

Comment:
Please remove this empty line in order to ensure the consistency in the code. (like in `deinit()`)